lib/presenter/pages/conversion_review/widgets/button_confirm_conversion.dart

```diff
@@ -0,0 +1,42 @@
+import '../../../../domain/entities/conversion_entity.dart';
+import '../../../controllers/transactions/all_transactions_provider.dart';
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class ButtonConfirmConversion extends HookConsumerWidget {
+  const ButtonConfirmConversion({
+    Key? key,
+    required this.conversionEntity,
+  }) : super(key: key);
+
+  final ConversionEntity conversionEntity;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final transactions = ref.watch(allTransactionsProvider.state);
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 45),
+      child: MaterialButton(
+        onPressed: () {
+          Navigator.pushNamed(context, '/conversion_confirmation');
```

Comment:
Quando for usar pushNamed é possível utilizar a rota definida no próprio código da página. Por exemplo, se há uma variável estática route definida seria possível escrever: ConversionConfirmation.route ou o nome da sua página.

Reply:
Isso já foi feito em outro card no futuro